bears/c_languages/CPPLintBear.py

```diff
@@ -26,12 +32,17 @@ class CPPLintBear:
     def create_arguments(filename, file, config_file,
                          max_line_length: int=79,
                          cpplint_ignore: typed_list(str)=(),
-                         cpplint_include: typed_list(str)=()):
+                         cpplint_include: typed_list(str)=(),
+                         indent_size: int=2,
+                         use_spaces: bool=True):
         """
```

Comment:
Is there any other way to check what kind of indent_size the user is using than relying on the user himself to give that information? Also does this fix it in case the indent_size is not matching or the user is using tabs?